routes/adminRoutes.js

```diff
@@ -14,7 +14,12 @@ router.post('/signIn', async (req, res) => {
 
     // Max age is in milliseconds.
     res.cookie("token", authData.token, { maxAge: authData.jwtExpirySeconds * 1000 })
-    res.end();
+    res.send({
+      username,
+      token: authData.token,
```

Comment:
Is there a reason for sending the token back in the body since it's already getting sent back as a token?